html/html-and-the-web_div-salad/index.html

```diff
@@ -67,5 +68,5 @@ <h3>Skillset</h3>
         </ul>
```

Comment:
I  can't find the opening tag to the unordered list and the list-items. Is there a list?